ui/src/components/Homepage/index.jsx

```diff
@@ -65,9 +65,8 @@ export default class HomePage extends Component {
           </Typography>
         </Hidden>
         <Typography variant="h5">
-          {
-            "A task execution framework that supports Mozilla's continuous integration and release processes."
-          }
+          &quot;A task execution framework that supports Mozilla&apos;s
+          continuous integration and release processes. &quot;
```

Comment:
This was fixed in https://github.com/taskcluster/taskcluster/commit/3da927a303f5e7062780f71bea6648f74626ccb9. The latest master branch should be good.